plugins/catalog-backend-module-gitlab/package.json

```
@@ -1,72 +1,76 @@
 {
-  "name": "@backstage/plugin-catalog-backend-module-gitlab",
-  "description": "A Backstage catalog backend module that helps integrate towards GitLab",
-  "version": "0.3.15",
-  "main": "src/index.ts",
-  "types": "src/index.ts",
-  "license": "Apache-2.0",
-  "publishConfig": {
-    "access": "public"
-  },
-  "exports": {
-    ".": "./src/index.ts",
-    "./alpha": "./src/alpha.ts",
-    "./package.json": "./package.json"
-  },
-  "typesVersions": {
-    "*": {
-      "alpha": [
-        "src/alpha.ts"
-      ],
-      "package.json": [
-        "package.json"
-      ]
-    }
-  },
   "backstage": {
     "role": "backend-plugin-module"
   },
-  "homepage": "https://backstage.io",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/backstage/backstage",
-    "directory": "plugins/catalog-backend-module-gitlab"
-  },
-  "keywords": [
-    "backstage"
-  ],
-  "scripts": {
-    "start": "backstage-cli package start",
-    "build": "backstage-cli package build",
-    "lint": "backstage-cli package lint",
-    "test": "backstage-cli package test",
-    "prepack": "backstage-cli package prepack",
-    "postpack": "backstage-cli package postpack",
-    "clean": "backstage-cli package clean"
-  },
+  "configSchema": "config.d.ts",
   "dependencies": {
     "@backstage/backend-common": "workspace:^",
     "@backstage/backend-plugin-api": "workspace:^",
     "@backstage/backend-tasks": "workspace:^",
     "@backstage/catalog-model": "workspace:^",
     "@backstage/config": "workspace:^",
     "@backstage/integration": "workspace:^",
+    "@backstage/plugin-catalog-common": "workspace:^",
     "@backstage/plugin-catalog-node": "workspace:^",
+    "@backstage/plugin-events-node": "workspace:^",
+    "@gitbeaker/rest": "^40.0.3",
```

```suggestion
    "@gitbeaker/rest": "^39.25.0",
```

if that's okay, to align with the other deps in the repo - this should clean up the yarn.lock diff